applications/callflow/src/module/cf_log_in_out_ring_group.erl

```diff
@@ -0,0 +1,91 @@
+%%%-------------------------------------------------------------------
+%%% @doc
+%%% Logs a user in or out of a ring group
+%%% @end
+%%% @contributors
+%%%   Max Lay
+%%%-------------------------------------------------------------------
+-module(cf_log_in_out_ring_group).
+
+-behaviour(gen_cf_action).
+
+-include("callflow.hrl").
+
+-export([handle/2]).
+
+-define(ON_VAL, 0).
+-define(DEFAULT_OFF_VAL, 66269664000).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%% Entry point for this module, attempts to call an endpoint as defined
+%% in the Data payload.  Returns continue if fails to connect or
+%% stop when successfull.
+%% @end
+%%--------------------------------------------------------------------
+-spec handle(kz_json:object(), kapps_call:call()) -> 'ok'.
+handle(Data, Call) ->
+    _ = kapps_call_command:answer(Call),
+    CallflowId = kz_json:get_value(<<"callflow_id">>, Data),
```

Comment:
If you use 'kz_json:get_ne_binary_value/2` it indicates the expected type and our AST parsers can infer the type for the JSON schema and add it.